### ISSUE TITLE
README found in getting-started needs updating

### ISSUE THREAD

**Issue by internetoftim** [gradient-ai/Graphcore-Pytorch#7]
Readme is from the original BERT-L finetuning README, this needs to be updated to talk about the 2 notebooks in get started:
- BERT-L
- Schnet

**Comment by nmb-paperspace** [comment]
This is part of inspecting and removing auxiliary files in the repos that I unfortunately didn't haven't have time to get to before launch. My solution would be remove the `README.md` and have the users use the top-level `README_first.ipynb`, since Paperspace Notebooks do not render `.md` files anyway, a known internal issue.
